src/config.js

```diff
@@ -1,6 +1,21 @@
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
 const path = require('path')
+const joi = require('joi')
+
+
+const environmentSchema = joi.object().keys({
+	DATABASE_URL: joi.string().required(),
+	PORT: joi.number().required(),
+	TOKEN_SECRET: joi.string().required(),
+	EMAIL_SERVICE_HOST: joi.string().required(),
+	EMAIL_SERVICE_PORT: joi.number().required(),
+	EMAIL_SERVICE_PASSWORD: joi.string().required(),
+	EMAIL_SERVICE_USERNAME: joi.string().required(),
+	EMAIL_FROM: joi.string().required(),
+	FRONTEND_BASE_URL: joi.string().required(),
```

Comment:
let's ensure its a valid url
```suggestion
	FRONTEND_BASE_URL: joi.string().uri().required(),
```